### DISCUSSION TITLE
Better interaction with image management, type management and matrix

### DISCUSSION THREAD

**Discussion by friebsch** [dainst/idai-field-client#169] [Ideas]
At the moment, image management, type management and matrix are opened in separate windows. In principle, however, all 3 would benefit from closer interaction with the resource view and the map view.

For example, one could display the images and types in the map area and thus filter directly on the basis of the selected resources, or filter the resource selection on the basis of the images.

The matrix would also interact well with the map and displayed resources. This could be done as an overlay on top of the map, so that you can select specific resources in the matrix. 

There are many options here that would help in working with and analysing the data. It also minimises context switching.

**Comment by lsteinmann** [comment]
See also: [possible future rework of the Matrix-tool](https://github.com/dainst/idai-field/milestone/1).

* The map could react to the stratigraphy and show only resources above or below the currently selected resource (also related to [the map](https://github.com/dainst/idai-field/milestone/2), obviously).  
* Matrix view could be combined with additional info like images etc. (see @friebsch s concept).
* Matrix could be used to restrict the map-view, or there could be a combination of both.
